Sources/AppAuth/iOS/OKTAuthorizationService+IOS.m

```diff
@@ -18,6 +18,10 @@
         Copyright (C) 2019 Okta Inc.
  */
 
+#import <Foundation/Foundation.h>
+
+#if !TARGET_OS_OSX
```

Comment:
Is it because this file is included to MacOS projects as well?

Reply:
Differentiating between macOS and iOS can be tricky, since we need to support both TARGET_OS_IOS and TARGET_OS_SIMULATOR. Since we don't need to worry about tvOS or watchOS, saying `!TARGET_OS_OSX` is the simplest expression here.